tests/integration/targets/inventory_vmware_vm_inventory/runme.sh

```diff
@@ -1,83 +1,57 @@
 #!/usr/bin/env bash
```

Comment:
```
ERROR: [Errno 13] Permission denied: b'./runme.sh'
```

Execution bit have to be set for this file, for running the test with `ansible-test integration --docker`: 
```sh
$ chmod +x tests/integration/targets/inventory_vmware_vm_inventory/runme.sh
```

Reply:
It is already set - 

```
-rwxr-xr-x  1 akasurde  admin   206 Apr  6 16:05 runme.sh
```

Reply:
I just did a fresh clone to test and still (100**644**). It should be (100**755**): 

```bash
$  git ls-files -s tests/integration/targets/inventory_vmware_vm_inventory/runme.sh
100644 b71b2448314b60c085c7716defc08277ab058223 0	tests/integration/targets/inventory_vmware_vm_inventory/runme.sh
```